app/build.gradle.kts

```diff
@@ -54,6 +54,7 @@ dependencies {
     implementation(Deps.Coroutines.android)
     implementation(Deps.multiplatformSettings)
     implementation(Deps.koinCore)
+    implementation(Deps.AndroidX.lifecycle_runtime)
```

Comment:
I thought this wouldn't need to get specified since it comes transitively with the other lifecycle libs, but I guess not.